server/db/driver/pg/accounts_online_test.go

```diff
@@ -89,6 +90,14 @@ func TestAccounts(t *testing.T) {
 		t.Fatal("accounts has unexpected data")
 	}
 
+	anAcct, err := archie.AccountInfo(accts[0].AccountID)
```

Comment:
Let's also test `PayAccount` before `PayAccount` to test the null cases.

Reply:
I  added a separate nullifying section.